src/Symfony/Component/Validator/Constraints/Unique.php

```diff
@@ -26,6 +26,7 @@ class Unique extends Constraint
 
     public array|string $fields = [];
     public ?string $errorPath = null;
+    public bool $multipleErrors = false;
```

Comment:
naming :) what about this? multipleErrors looks like a consequence of something, not an option
```suggestion
    public bool $quickCheck = true;
```

Reply:
The current name doesn't seem right to me. It's about having all errors, not necessarily multiple ones.

Maybe `$allErrors`?

If we want to name it closer to the behavior, the current one is "stopping on first error", which is more or less similar to PHPUnit `--no-on-error` flag.

So, another suggestion might be: `$stopOnFirstError`?

Reply:
Maybe "$all" so all must be unique?

Reply:
Agreed with `$stopOnFirstError` :+1: 

The PR is updated in that way + `CHANGELOG.md`